src/goto-programs/restrict_function_pointers.cpp

```diff
@@ -24,6 +24,11 @@ void typecheck_function_pointer_restrictions(
   const goto_modelt &goto_model,
   const function_pointer_restrictionst &restrictions)
 {
+  const std::string options =
+    "--" RESTRICT_FUNCTION_POINTER_OPT "/"
+    "--" RESTRICT_FUNCTION_POINTER_FROM_FILE_OPT "/"
+    "--" RESTRICT_FUNCTION_POINTER_BY_NAME_OPT;
```

Comment:
:question: Is it not possible to resitrct by name from file?

Reply:
Since this is constant - perhaps worth declaring as global constant and then not having to pass into each of the functions

Reply:
Currently that's not supported as the from file option was mostly intended for consuming the output from goto-analyzer which will refer to the calls via indices rather than names. Though it would also make sense to let a user supply manually written json files at some point.

Reply:
This very string is only used in this method.